GitUI/CommandsDialogs/FormBrowse.InitMenusAndToolbars.cs

```diff
@@ -137,17 +137,35 @@ void WorkaroundToolbarLocationBug()
             }
         }
 
+        private void UpdateTooltipWithShortcut(ToolStripItem button, Command command)
+            => UpdateTooltipWithShortcut(button, GetShortcutKeys(command));
+
+        private void UpdateTooltipWithShortcut(ToolStripItem button, Keys keys)
+            => UpdateTooltipWithShortcut(button, keys.ToShortcutKeyToolTipString());
+
+        private void UpdateTooltipWithShortcut(ToolStripItem button, string shortcut)
+        {
+            if (string.IsNullOrWhiteSpace(shortcut))
+            {
+                return;
+            }
+
+            int indexShortcut = button.ToolTipText.LastIndexOf('(');
```

Comment:
What if the shortcut is / contains '('? Rather use a "fixed space" Unicode character as delimiter.